### PR TITLE
docs(modeler): clearly scope token simulation purpose

### PR DESCRIPTION
**Pull request by nikku**
## Description

This PR updates the token simulation documentation as per [this feedback](https://camunda.slack.com/archives/C032H77434N/p1702032295163309?thread_ts=1702022400.860369&cid=C032H77434N).

We don't want to delude users into thinking that token simulation simulates Camunda execution. In fact it simulates BPMN execution, and is a BPMN learning tool, not a Camunda learning tool. It does not intend to simulate Camunda, nor to be a full BPMN 2.0 engine.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/modeler/web-modeler/token-simulation.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-description: You can use the token simulation feature to observe how the process will behave when it is executed.
+description: Use token simulation to get a deeper understanding of the BPMN execution semantics by playing, pausing, and resetting your simulation.
 ---
 
-You can use the token simulation feature to observe how the process will behave when it is executed.
+Use token simulation to get a deeper understanding of the BPMN execution semantics by playing, pausing, and resetting your simulation.
 
 ### Turn on/off
 
+First, open your BPMN diagram in Modeler.
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/token-simulation.md:11]: Token simuation is only available in the Design mode

File: docs/components/modeler/web-modeler/token-simulation.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
 ---
 id: token-simulation
 title: Token simulation
-description: You can use the token simulation feature to observe how the process will behave when it is executed.
+description: Use token simulation to get a deeper understanding of the BPMN execution semantics by playing, pausing, and resetting your simulation.
 ---
 
-You can use the token simulation feature to observe how the process will behave when it is executed.
+Use token simulation to get a deeper understanding of the BPMN execution semantics by playing, pausing, and resetting your simulation.
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/token-simulation.md:7]: A note that behavior in token simulation may differ from engine execution would be valuable, with some examples (e.g. tokens look different, no implementation details are considered so link events are uncorrelated)

To see how the process would execute on a Zeebe cluster, use Play instead

**Review by HanselIdes — COMMENTED** [review]
I added a few comments. Given Christina's question about 'how does this compare to Play', I think the documentation needs to explicitly compare the purpose and behavior of token simulation to Play.

File: docs/components/modeler/web-modeler/token-simulation.md (hunk 1 above)
Comment (nikku) [reply docs/components/modeler/web-modeler/token-simulation.md:11]: There may be other things broken in this document. I'm not touching the document up completely, but rather focus on clearly scoping the purpose.

Let's follow-up with everything else in a dedicated PR.

File: docs/components/modeler/web-modeler/token-simulation.md [hunk 3]
```diff
@@ -1,13 +1,25 @@
 ---
 id: token-simulation
 title: Token simulation
-description: You can use the token simulation feature to observe how the process will behave when it is executed.
+description: Use token simulation to get a deeper understanding of the BPMN execution semantics by playing, pausing, and stepping through your processes.
```
Comment (nikku) [inline docs/components/modeler/web-modeler/token-simulation.md:4]: > by playing, pausing, and stepping through your processes

@christinaausley Could you elaborate what this contributes to the description?

Comment (christinaausley) [reply docs/components/modeler/web-modeler/token-simulation.md:4]: Correct!

File: docs/components/modeler/web-modeler/token-simulation.md (hunk 1 above)
Comment (nikku) [reply docs/components/modeler/web-modeler/token-simulation.md]: Addressed via https://github.com/camunda/camunda-docs/pull/3011/commits/e7989c7e710ef136b7985561a8fc16eb99a9d7ba, now clearly stating that it is in Design mode.

**Comment by christinaausley** [comment]
> Additionally, I wanted to play around with this. It seems the gifs are out of date, and I'm not sure where token simulation exists now that the UI has been updated. Perhaps out of scope for this PR, but we may need to update these instructions for the user and update with screenshots. I find gifs are a challenge to maintain, and aren't the best for accessibility purposes.

So long as we have another issue in the backlog to clean this up, I'm good with this. Thanks for updating 😄

**Review by christinaausley — APPROVED** [review]

**Pull request merged**